projects/sandbox/src/app/chart/pie/pie.component.ts

```diff
@@ -119,7 +127,12 @@ export class PieComponent implements AfterContentInit {
             display: false
           },
           datalabels: {
-            color: 'white',
+            color: (context: { dataIndex: number }): string => {
+              if (context.dataIndex > 3) {
```

Comment:
what means dataIndex > 3?

Reply:
suggestion: write a function that checks this condition `e.g. isContextValid()` is easy to know what is evaluating.

Reply:
I've used the `Context` import from `chartjs-plugin-datalabels` and added a comment.

> *branch name `eat` instead of `feat` joy tested with the actual mock, x *it sorts ASC, DESC, of disables works heavy_check_mark *the search function seems is not working. x ![afbeelding](https://user-images.githubusercontent.com/19560266/243329559-fe458500-f843-4f7b-acd0-cb33527dd595.png)

I've renamed the branch and created [another PR here](https://github.com/europeana/metis-ui/pull/773).